keras_nlp/models/roberta/roberta_preprocessor.py

```diff
@@ -236,6 +241,8 @@ def __init__(
         truncate="round_robin",
         **kwargs,
     ):
+        assert_tf_text_installed(self.__class__.__name__)
```

Comment:
Does this surface a readable error message? Is this the only place we need to do this?

Reply:
If you search code, we do this in already 8 other places, this is a spot we missed. And yes, this just raises an error, utility so our language could stay consistent.

https://github.com/keras-team/keras-nlp/blob/87019700821662d33d573ce47a1348f44418aa5a/keras_nlp/utils/tf_utils.py#L69-L75